src/collection.js

```diff
@@ -308,15 +357,25 @@ export default class Collection {
     if (!record.id) {
       throw new Error("A record id is required.");
     }
-    const reqOptions = this._collOptions(options);
-    const { permissions } = reqOptions;
+    const { permissions } = options;
+    let updateOptions = {
+      headers: this._getHeaders(options),
+      safe: this._getSafe(options),
+    };
+    const { last_modified } = { ...record, ...options };
+    if (last_modified) {
+      updateOptions = { ...updateOptions, last_modified };
+    }
+    if (options.patch) {
+      updateOptions = { ...updateOptions, patch: options.patch };
+    }
```

Comment:
Nit: this may just work:

```js
const { last_modified } = { ...record, ...options };
const updateOptions = {
  headers: this._getHeaders(options),
  safe: this._getSafe(options),
  last_modified,
  patch: !!options.patch,
};
```